lib/typeahead_search.rb

```diff
@@ -21,8 +23,10 @@ def xapian_search
     begin
       xapian_search = run_query
     rescue ActsAsXapian::UnhandleRuntimeError => e
-      # Wildcard expands to too many terms
-      Rails.logger.info "Wildcard query '#{query}' caused: #{e.message.force_encoding('UTF-8')}"
```

Comment:
I think we should keep the log output as well as the optional exception notification.